packages/common/validation.js

```diff
@@ -28,3 +32,28 @@ export const alphaNumeric = value =>
 // Phone number validation
 export const phoneNumber = value =>
   value && !/^(0|[1-9][0-9]{9})$/i.test(value) ? 'Invalid phone number, must be 10 digits' : undefined;
+
+export const validateForm = (fValues, fSchema) => {
+  let errors = {};
+  const validateForm1 = (values, schema, collector) => {
```

Comment:
Names must be meaningful, why the name `validateForm1` here, it does not have meaning

Reply:
That is an inner function and not visible from outside, that's why it was called in such way. Anyways, I have renamed it.

Reply:
@evgenijolshanskij It doesn't matter, names must be meaningful at all times